CHANGELOG.md

```diff
@@ -1,39 +1,40 @@
+## 1.0.1
+
+- Remove unnecessary `clock` dependency and use the built-in `Stopwatch` object
+  instead.
+
 ## 1.0.0+5
 
 - Update and test against Dart 2.17.
 
-[More details available in
-GitHub](https://github.com/llucax/pausable_timer/milestone/9?closed=1).
+[More details available in GitHub](https://github.com/llucax/pausable_timer/milestone/9?closed=1).
```

Comment:
Again, could you avoid unnecessary chances in formatting of the rest of the file?

Reply:
What is wrong with the formatting?

Reply:
Copied from #49: There is nothing wrong with the formatting, but there is nothing wrong with how it was formatted before either, so I don't see any reason to change it and add noise to the diff. It is distracting and maybe the next person that opens a PR thinks there is a better formatting too (or has another tool to automatically format it). It just makes no sense to change it, unless it is a standard tool that is making the auto formatting and we can add a check to the CI to make sure the same format is always used (as it is with the dart code).